### PR TITLE
add static home page

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+
 import Product from './pages/product/product';
 function App() {
   return (
     <div className="App">
       <Router>
       <Navbar/>
         <Routes>
+
+          <Route path='/home' element={<Home/>}/>
```
Comment (MiheerTamkhane) [inline src/App.js:16]: path='/' will work fine

**Review by MiheerTamkhane — APPROVED** [review]
Other than that code is good to merge

File: src/App.js (hunk 1 above)
Comment (harshkotkar5557) [reply src/App.js:16]: okay I will that.

**Review by Mahi24-prog — APPROVED** [review]
Looks good

**Pull request merged**